l10n_it_pec/views/partner_view.xml

```diff
@@ -0,0 +1,40 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+    <record id="view_partner_pec_form" model="ir.ui.view">
+        <field name="name">res.partner.pec.form</field>
+        <field name="model">res.partner</field>
+        <field name="inherit_id" ref="base.view_partner_form"/>
+        <field name="arch" type="xml">
+            <field name="website" position="before">
```

Comment:
The way you suggest isn't applicable due to a change in the commit of mail/res_partner_views.xml [()](https://github.com/OCA/OCB/commit/59b4836e8d14abdca49e43cdd72b2151e78230d4#diff-2db613da90ddb55611faa104bec4752a) that placed the email element inside a div.
I had to use xpath to place the email_pec element in the right position.
Thanks for your suggestion.

Reply:
Great! :+1: Since you are supposing the changes of 'mail.res_partner_view_form_inherit_mail', you should inherit from that view, and from 'mail' module, otherwise the inheritance chain might break

Reply:
Ok, thanks for your suggestion. I changed both.

Reply:
@mymage It seems good! 

@monen17 what do you think?